### DISCUSSION TITLE
Support for multiple URLs with Webhook plugin

### DISCUSSION THREAD

**Discussion by piscopoc** [emqx/emqx#6464] [Ideas]
Does the webhook plugin support multiple urls? Wondering if that specific plugin supports options for multiple webhooks or is it just limited to a single endpoint:

Something like this:
```
web.hook.api1.url = http://127.0.0.1:8080/endpoint1
web.hook.api2.url = http://127.0.0.1:8080/endpoint2
```

**Comment by zmstone** [comment]
converting to a discussion.

**Comment by zmstone** [comment]
Hi @piscopoc 
No, as of today, it supports only one URl.
Maybe use a load balancer ?

**Reply by piscopoc** [reply]
Thanks, appreciate the quick response!

**Reply by JiaweiGithub** [reply]
We have the same needs, are there any follow-up plans?
